automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/methodrepo/MethodRepoSpec.scala

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.dsde.firecloud.test.methodrepo
+
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, AuthTokens, Config}
+import org.broadinstitute.dsde.firecloud.fixture.MethodData
+import org.broadinstitute.dsde.firecloud.page.Table
+import org.broadinstitute.dsde.firecloud.page.methodrepo.{CreateMethodModal, MethodRepoPage}
+import org.broadinstitute.dsde.firecloud.test.WebBrowserSpec
+import org.scalatest._
+
+class MethodRepoSpec extends FreeSpec with WebBrowserSpec with Matchers {
+
+  implicit val authToken: AuthToken = AuthTokens.harry
+
+  "A user" - {
+    "should be able to create a method, see it, and then redact it" in withWebDriver { implicit driver =>
```

Comment:
Creating a method and redacting a method should be 2 separate tests. The redact test can create the method with a web service call instead of driving the UI again since that will be exercised in the create method test.